components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -78,21 +80,16 @@ function InputBox(props: InputBoxProps) {
     }
   }
 
-  const querySubmitted = React.useRef(false)
-
   const maybeAutofocus = (node: HTMLTextAreaElement | null) => {
     if (!node) {
```

Comment:
I'm a bit curious about how often this gets called - is it every render, or is it only when the ref (`node`) is changed?